tests/dragonfly/sentinel_test.py

```diff
@@ -127,7 +131,7 @@ async def test_failover(df_local_factory, sentinel):
 
     master_client = aioredis.Redis(port=master.port)
     replica_client = aioredis.Redis(port=replica.port)
-    print("master: " + str(master.port) + " replica: " + str(replica.port), flush=True)
+    log("master: " + str(master.port) + " replica: " + str(replica.port))
```

Comment:
I have invoked this branch on CI - and doesn't see this log line:
https://github.com/dragonflydb/dragonfly/actions/runs/4290749525/jobs/7475092774
Can you please check as this is the main motivation for this PR?

Reply:
By default Pytest captures all outputs and prints them only on test failure. However, it can be disabled in two ways:
1. Globally, by passing a flag such as `-s` which disables all captures across all tests
2. Per-test, using `capsys`:
```python3
def my_test(capsys):
    print("This won't show")
    with capsys.disabled():
        print("This will show")
```

There is a bit of info [here](https://docs.pytest.org/en/7.1.x/how-to/capture-stdout-stderr.html), but this is the main idea.

I think using `-s` and disabling capturing globally isn't a great idea.

Reply:
But we do run with -s in CI
https://github.com/dragonflydb/dragonfly/actions/runs/4290749525/workflow#L55

That is why I am wondering.

Can you please make sure that you are able to observe the logs in CI upon a failure? 
Thanks!

Reply:
OK, I have found the problem. Using `-s` disables Pytest from hiding `print` calls, and instead makes them print directly to the console.

Behind the scenes, `logging` is a bit different, and Pytest **always** inserts some of its code into it. So, using `-s` won't affect that. Instead, you should use [live logs](https://docs.pytest.org/en/7.1.x/how-to/logging.html#live-logs). For example, the following line of code will achieve your desired output:
```shell
pytest -xv dragonfly/sentinel_test.py --log-cli-level=INFO
```

Here is where I'd like your input. It is possible to add to the config an option which enables live logs for all tests. However, this means that all logs (**not** `print`s) done in tests will always be printed to the console, even if they pass. Instead, it is also possible to just revert to using `print`s and `-s`.

Is this a desired behavior or not? Let me know.

Reply:
If I understand correctly it makes the behaviour for logging and printing consistent, which is exactly what we are after right now.
If we decide some day that we are too verbose we can revisit.
So yea "enable live logs for all tests." sounds good to me.